### ISSUE TITLE
Can not sign in with Google

### ISSUE THREAD

**Issue by fan123199** [supabase/supabase-flutter#1041]
**Describe the bug**

I am using supabase_flutter: 2.6.0 in package-lock.yaml

And using [instrucion code](https://supabase.com/docs/guides/auth/social-login/auth-google?queryGroups=platform&platform=flutter#ios-and-android) to implement our app.

It is reported yesterday by our user that they cannot login, and then I test on Android 15 beta (Google Pixel 6 Pro) and Android 14 (Oneplus 12).  It really cannot login, error shows

```
 [GoogleAuthUtil] [GoogleAuthUtil] error status:UNKNOWN with method:getTokenWithDetails
```

And I search from stackoverflow, https://stackoverflow.com/questions/79009273/flutter-platformexception-when-trying-to-get-google-sign-in-access-token, The fix works.

 But it should be fixed in supabase docs.  It should add scope to Google sign in function params.

**Comment by elliottetzkorn** [comment]
I had the same issue all of a sudden, and had to spend a whole weekend figuring out this fix. Something must have just changed on Google's end, very annoying

**Comment by fan123199** [comment]
> I had the same issue all of a sudden, and had to spend a whole weekend figuring out this fix. Something must have just changed on Google's end, very annoying

Yes, I think Google really make a bug. This should not happened. It is a production incident.

**Comment by elliottetzkorn** [comment]
The scopes issue fixes debug but from what I can tell production is totally destroyed. I cannot login via the release signing sha no matter what I do. Is anyone seeing that too??

**Comment by dshukertjr** [comment]
It seems like Google has fixed the issue now internally?
https://github.com/flutter/flutter/issues/155429
